dev/a11y_assessments/test/accessibility_guideline_test.dart

```diff
@@ -24,7 +24,11 @@ void main() {
       await expectLater(tester, meetsGuideline(textContrastGuideline));
       await expectLater(tester, meetsGuideline(androidTapTargetGuideline));
       await expectLater(tester, meetsGuideline(iOSTapTargetGuideline));
-      await expectLater(tester, meetsGuideline(labeledTapTargetGuideline));
+      // Skip the Slider use case, it uses tap action without label on purpose.
+      // see ttps://github.com/flutter/flutter/pull/157601.
+      if (useCase.name != 'Slider') {
```

Comment:
Not very sure whether we can modify the test like this. @hannah-hyj and @chunhtai are OOO, we can double check with them next week:)

Reply:
I'm also not sure but the this rule guideline test doesn't really work for Slider. When testing the Android components Slider, it is tappable in a11y mode without changing the Slider value. 

Semantic with tap action require label but an existing Slider [test](https://github.com/flutter/flutter/pull/102129) prevents this. So i decided to skip the a11y test itself.

Reply:
I see. If the current behavior is intended, I think we can create an issue for the guideline and add a TODO here:) 
CC: @chunhtai

Reply:
This doesn't look right, the slider use case does assign a label to the slider, it should not have failed the test. Can you dump the semantics tree when run the a11y_assessment app? i suspect something is forming a node unexpectedly

Reply:
@chunhtai 
I've reverted the a11y_assessment. changes to show the error logs in the CI to access.

https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8732575348927856369/+/u/run_test.dart_for_framework_tests_shard_and_subshard_misc/stdout

Reply:
So I try to understand the issue. It looks like ios that it will issue regular tap if there isn't a semantics tap. the regular tap taps the center of the slider causes it to change the value to the middle.

The code here is a workaround so that the ios thinks there is a semantics tap so that it won't issue regular tap. This seems to me it is something that we should fix in iOS embedding. More specifically here should return YES if this is a slider
https://github.com/flutter/engine/blob/e553cd61645fc9bb21e62055e75ccddb09b596eb/shell/platform/darwin/ios/framework/Source/SemanticsObject.mm#L695

Reply:
Tagging @bleroux who can help fix this in the engine side.

Reply:
If we agree the fix should be on the iOS side, I think we should close this pr